nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-tomcat/mysql-quartz-cluster/bin/create-quartz-tables.sql

```diff
@@ -0,0 +1,185 @@
+#
+# In your Quartz properties file, you will need to set
+# org.quartz.jobStore.driverDelegateClass = org.quartz.impl.jdbcjobstore.StdJDBCDelegate
+#
+#
+# By: Ron Cordell - roncordell
+#  I didnt see this anywhere, so I thought Id post it here. This is the script from Quartz to create the tables in a MySQL database, modified to use INNODB instead of MYISAM.
+
+DROP TABLE IF EXISTS QRTZ_FIRED_TRIGGERS;
+DROP TABLE IF EXISTS QRTZ_PAUSED_TRIGGER_GRPS;
+DROP TABLE IF EXISTS QRTZ_SCHEDULER_STATE;
+DROP TABLE IF EXISTS QRTZ_LOCKS;
```

Comment:
Under Linux it does not work because of identifier sensitivity. After applying successfully the sql script there is an error on startup with "Table 'nuxeo.qrtz_LOCKS' doesn't exist" while QRTZ_LOCKS has been created. Windows and OSX look like to have case insensitive table name.